services/web/server/src/simcore_service_webserver/conversations/_socketio.py

```diff
@@ -62,16 +97,79 @@ async def _send_message_to_recipients(
 ):
     async for _ in limited_as_completed(
         (
-            send_message_to_user(
-                app, recipient, notification_message, ignore_queue=True
-            )
+            send_message_to_user(app, recipient, notification_message)
             for recipient in recipients
         ),
         limit=_MAX_CONCURRENT_SENDS,
     ):
         ...
 
 
+async def notify_conversation_created(
+    app: web.Application,
+    *,
+    recipients: set[UserID],
+    project_id: ProjectID,
+    conversation: ConversationGetDB,
+) -> None:
+    notification_message = SocketMessageDict(
+        event_type=SOCKET_IO_CONVERSATION_CREATED_EVENT,
+        data={
+            **ConversationCreatedOrUpdatedEvent(
+                project_id=project_id,
+                **conversation.model_dump(),
```

Comment:
TIP: exclude_none or exclude_uset on dump?

Reply:
Not necessary here, I revalidate using the output model.